app/views/idv/welcome/show.html.erb

```diff
@@ -6,20 +6,37 @@
     ) do %>
 <%= render PageHeadingComponent.new.with_content(@title) %>
   <p>
-    <%= t(
-          'doc_auth.info.getting_started_html',
-          sp_name: @sp_name,
-          link_html: new_tab_link_to(
-            t('doc_auth.info.getting_started_learn_more'),
-            help_center_redirect_path(
-              category: 'verify-your-identity',
-              article: 'how-to-verify-your-identity',
-              flow: :idv,
-              step: :welcome,
-              location: 'intro_paragraph',
+    <% if decorated_sp_session.selfie_required? then %>
```

Comment:
I am thinking ahead to how we will determine if a user is stepping up. Do you think it is time for us to put together a `Idv::WelcomePresenter` to encapsulate some of this view logic (e.g. the sp_name, title, and this paragraph's translation)?

Reply:
That strikes me as an excellent suggestion. Would you like me to add it to this ticket, or open a new one?

Reply:
Moving the ticket back to in-progress and adding WelcomePresenter.